resources/views/front-end/photo_album.blade.php

```diff
@@ -0,0 +1,125 @@
+        </div>
+        <!-- Default (PhotoSwipeUI_Default) interface on top of sliding area. Can be changed. -->
+        <div class="pswp__ui pswp__ui--hidden">
+            <div class="pswp__top-bar">
+                <!--  Controls are self-explanatory. Order can be changed. -->
+                <div class="pswp__counter"></div>
+                <button class="pswp__button pswp__button--close" title="Close (Esc)"></button>
+                <button class="pswp__button pswp__button--share" title="Share"></button>
+                <button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>
+                <button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>
+                <!-- Preloader demo http://codepen.io/dimsemenov/pen/yyBWoR -->
+                <!-- element will get class pswp__preloader--active when preloader is running -->
+                <div class="pswp__preloader">
+                    <div class="pswp__preloader__icn">
+                      <div class="pswp__preloader__cut">
+                        <div class="pswp__preloader__donut"></div>
+                      </div>
+                    </div>
+                </div>
+            </div>
+            <div class="pswp__share-modal pswp__share-modal--hidden pswp__single-tap">
+                <div class="pswp__share-tooltip"></div> 
+            </div>
+            <button class="pswp__button pswp__button--arrow--left" title="Previous (arrow left)">
+            </button>
+            <button class="pswp__button pswp__button--arrow--right" title="Next (arrow right)">
+            </button>
+            <div class="pswp__caption">
+                <div class="pswp__caption__center"></div>
+            </div>
+        </div>
+    </div>
+</div>
+
+    <div class="card">
+        <div class="card-body">
+            <div class="row d-flex align-items-stretch align-items-center">
+                @foreach ($thumbnails as $id=>$thumbnail)
+                    <img id="thumbnail" class="col-*-* px-1" src="{!! $thumbnail !!}" onclick="openGallery('{{ $loop->index }}');">
+                @endforeach
+            </div>
+            
+            <div class="mt-2">
+                <h5>Foto's toevoegen </h5>
+                <label class="input-group-btn">
+                    <span class="btn btn-primary">
+                        Browse&hellip; <input style="display: none;" class="form-control" type="file" id="file-select" name="photos[]" multiple required/>
+                    </span>
+                </label>
+                <input id="filesSelected" type="text" class="form-control" readonly>
+                <button class="btn btn-primary" id='submit' onclick="uploadPhoto()">Toevoegen</button>
+            </div>
+        </div>   
+    </div>
+</div>
+<style>
+#thumbnail {
+    width: auto;
+    height: auto;
+    padding-bottom: 1%;
+    min-width:33.33%;
+    max-width: 33.33%;
+    max-height: 300px;
+}
+</style>
```

Comment:
Zou je dit kunnen verplaatsen naar de scss files? En een ID voor thumnail lijkt me ook niet helemaal kloppen? Er zijn er namelijk meerder van op een pagina denk ik?

Mss ook even kijken of er goede bootstrap classes voor zijn.

Reply:
Dit is opgelost met bootstrap en blade, zie commit 
46d7899